roles/default/kiali-deploy/templates/openshift/clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
can you rename this so it is more specific - in [a similar way we have specific names for some role binding templates](https://github.com/kiali/kiali-operator/tree/master/roles/default/kiali-deploy/templates/openshift) this is for supporting oauth so name this `clusterrolebinding-oauth.yaml`.